srv_fai_config/hooks/install.SEAPATH

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Did you test that the `install` task was early enough to hook in order not to trigger the error?
Shouldn't we hook the `partition` task to be more explicit?

Reply:
The error occurs at the beginning of the install task (when the partitions are wiped / recreated).
The install hook happens just before the task so I think it's fine.
Am I missing something ?

Reply:
Ok that's great. I wasn't sure when the install task was called by FAI because the documentation wasn't that precise. LGTM.